### PR TITLE
Add .editorconfig to scaffold plugin, plugin-tests and package-tests.

### PR DESCRIPTION
**Pull request by miya0001**
Add `.editorcofig` to `wp scaffold plugin`, `wp scaffold plugin-tests`, `wp scaffold package-tests`.

https://core.trac.wordpress.org/browser/trunk
https://core.trac.wordpress.org/ticket/26429
http://editorconfig.org/

### CODE REVIEW

**Comment by danielbachhuber** [comment]
Hm. I thought I left a comment on this, but I guess I didn't.

`wp scaffold plugin` seems reasonable, but I'm not sure why `.editorconfig` should be added for `wp scaffold plugin-tests` and `wp scaffold package-tests`. Can you clarify?

**Comment by miya0001** [comment]
> I'm not sure why .editorconfig should be added for wp scaffold plugin-tests and wp scaffold package-tests

Ah, certainly... :)
Fixed.

File: php/commands/scaffold.php [hunk 1]
```diff
@@ -530,8 +530,11 @@ function plugin( $args, $assoc_args ) {
 			"$plugin_dir/Gruntfile.js" => Utils\mustache_render( 'plugin-gruntfile.mustache', $data ),
 		), $force );
 
+		$wp_filesystem = $this->init_wp_filesystem();
+		$wp_filesystem->copy( WP_CLI_ROOT . "/templates/.editorconfig", "$plugin_dir/.editorconfig", true );
```
Comment (danielbachhuber) [inline php/commands/scaffold.php]: Can you move this into the `create_files()` call above?

**Comment by danielbachhuber** [comment]
@miya0001 just one comment to address.

**Comment by miya0001** [comment]
> Can you move this into the create_files() call above?

Done! :smile:

**Pull request merged**